src/index.jsx

```diff
@@ -35,14 +40,19 @@ export async function prerender() {
 		// fetch latest release data
 		const { default: releaseLambda } = await import('./lambda/release.js');
 		const { version, url } = await (await releaseLambda()).json();
-		globalThis.prerenderPreactVersion = version;
-		globalThis.prerenderPreactReleaseUrl = url;
+		prerenderData.preactVersion = version;
+		prerenderData.preactReleaseURL = url;
+
+		// fetch latest stargazer count
+		const { default: repoLambda } = await import('./lambda/repo.js');
+		const { stargazers_count: stargazersCount } = await (await repoLambda()).json();
+		prerenderData.preactStargazers = stargazersCount;
```

Comment:
As this is very far from crucial (only used in a demo at the very bottom of the home page), I had originally left it out & we just used a hard-coded fallback. Now that this PR introduces a better prerender data setup, however, I felt like it was worth fetching on build & ditching the old hard-coded count.